storage/imx6ull-flashnor/Makefile

```diff
@@ -4,10 +4,18 @@
 # Copyright 2021 Phoenix Systems
 #
 
-NAME := imx6ull-flashnor
-LOCAL_SRCS := flashnor.c flashnor-ecspi.c flashnor-qspi.c flashnor-drv.c nor/nor.c
-LOCAL_HEADERS := flashnor-ecspi.h flashnor-qspi.h flashnor-drv.h nor/nor.h
+
+NAME := libflashnordrv-imx6ull
+LOCAL_SRCS := flashnor-ecspi.c flashnor-qspi.c flashnor-drv.c nor.c
+LOCAL_HEADERS := imx6ull-flashnor-ecspi.h imx6ull-flashnor-qspi.h imx6ull-flashnor-drv.h imx6ull-nor.h
 DEP_LIBS := libimx6ull-ecspi libimx6ull-qspi
+
+include $(static-lib.mk)
+
+NAME := imx6ull-flashnor
+LOCAL_SRCS := flashnor-srv.c
+LOCAL_HEADERS := 
```

Comment:
variables are automatically reset in `static-lib.mk` (https://github.com/phoenix-rtos/phoenix-rtos-build/blob/master/makes/static-lib.mk#L112), no need to explicitly empty them